include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/processOutputImageAndReport.kt

```diff
@@ -84,17 +85,23 @@ fun processOutputImageAndReport(
         bufferedImageType = recordOptions.pixelBitConfig.toBufferedImageType()
       )
     }
+
+    // Only used by CaptureResult.Changed
+    var diffPercentage by Delegates.notNull<Float>()
```

Comment:
I just want to know this: the reason you use `Delegates.notNull<Float>()` instead of `var diffPercentage = NaN` is that you want to trigger an error when the value is not set, right?

Reply:
Yes, I think it is a little more robust to force the developer to set the `diffPercentage` in each case separately than to set a default and only change it in the cases where it should be different. But it is mostly a matter of taste, in my opinion. I can change it if you wish.